docs/volumes.md

```diff
@@ -52,4 +52,117 @@ Additional volume disks are exposed in the alphabet order starting from b, so vo
 On pod remove expected all volumes and snapshot related to VM should be removed.
 
 ## Persistent Storage
-TODO
+## Flexvolume libvirt driver
+**NOTE: Currently, blocked by qemu connectivity issue, see details in !!!**
```

Comment:
Oh, forgot about issue for connectivity, wanted to add link here to it, will create and update.